spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -1075,155 +1080,16 @@ def _shellwidget_deleted(self, client):
         except RuntimeError:
             pass
 
-    def _create_client_for_kernel(self, connection_file, hostname, sshkey,
```

Comment:
Here I simply moved this method to the `Public API` section and made the previous `create_client_for_kernel` private.

Reply:
@ccordoba12 I think this method is being used on `spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py::test_tab_rename_for_slaves` so a change to the test needs to be done for the test to pass.

Reply:
Yep, it was used in a bunch of tests. I replaced that with the new API method.